contrib/jenkins_tests/rpm.sh

```diff
@@ -88,5 +88,40 @@ if [ $opt_checkpkg -eq 1 ]; then
     test_id=$((test_id+1))
 fi
 
+
+# check if we have email of indevidual users in the packages rpm/deb metadata Maintainer field
+pacakges_location="$rpm_dir"/dist-pkg/packages
+email_log_file="$rpm_dir"/dist-pkg/email_scan.log
+
+if [ $opt_rpm -eq 1 ]; then
+    search_filter="*.rpm"
+    test_info_exec="rpm -qpi --changelog"
+else
+    search_filter="*.deb"
+    test_info_exec="apt info"
+fi
+
+# iterate on all packages and extarct the metadata to outout file
+find "$pacakges_location" -type f -name "$search_filter" -exec $test_info_exec {} \; |& tee -a "$email_log_file"
+
+do_archive "$email_log_file"
+
+set +e
```

Comment:
Because i cant fail the grep command there is a meaning of rc 1 or 0 that i am checking later on.
The logic of "knowing" if step should fail depends on rc + output of that command